src/OrchardCore.Modules/OrchardCore.Workflows/Http/Controllers/HttpWorkflowController.cs

```diff
@@ -21,6 +22,7 @@ public class HttpWorkflowController : Controller
         private readonly IWorkflowStore _workflowStore;
         private readonly IActivityLibrary _activityLibrary;
         private readonly ISecurityTokenService _securityTokenService;
+        private readonly IStringLocalizer T;
```

Comment:
Please use `S` instead, we already use this convention everwhere

Reply:
For my own reference, when I contribute in future, if I see a location using `T` should I convert it to `S`?

Reply:
We use `S` for `IStringLocalizer`, `H` for `IHtmlLocalizer` and `T` for localizers in views only